File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Cpls/CPL0000Q00LaySubHangmogListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Cpls
{
    [Serializable]
	public class CPL0000Q00LaySubHangmogListItemInfo
	{
		private String _gumsaName;
		private String _resultDate;
		private String _cplResult;

		public String GumsaName
		{
			get { return this._gumsaName; }
			set { this._gumsaName = value; }
		}

		public String ResultDate
		{
			get { return this._resultDate; }
			set { this._resultDate = value; }
		}

		public String CplResult
		{
			get { return this._cplResult; }
			set { this._cplResult = value; }
		}

		public CPL0000Q00LaySubHangmogListItemInfo() { }

		public CPL0000Q00LaySubHangmogListItemInfo(String gumsaName, String resultDate, String cplResult)
		{
			this._gumsaName = gumsaName;
			this._resultDate = resultDate;
			this._cplResult = cplResult;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/cpls/CPL0000Q00LaySubHangmogListItemInfo.java
package nta.med.data.model.ihis.cpls;

import java.sql.Timestamp;

public class CPL0000Q00LaySubHangmogListItemInfo {
	private String gumsaName;
    private Timestamp resultDate;
    private String cplResult;
	public CPL0000Q00LaySubHangmogListItemInfo(String gumsaName,
			Timestamp resultDate, String cplResult) {
		super();
		this.gumsaName = gumsaName;
		this.resultDate = resultDate;
		this.cplResult = cplResult;
	}
	public String getGumsaName() {
		return gumsaName;
	}
	public void setGumsaName(String gumsaName) {
		this.gumsaName = gumsaName;
	}
	public Timestamp getResultDate() {
		return resultDate;
	}
	public void setResultDate(Timestamp resultDate) {
		this.resultDate = resultDate;
	}
	public String getCplResult() {
		return cplResult;
	}
	public void setCplResult(String cplResult) {
		this.cplResult = cplResult;
	}
}
